katz/src/test/kotlin/katz/data/IorTest.kt

```diff
@@ -132,5 +133,26 @@ class IorTest : UnitSpec() {
 
         }
 
+        "Ior.monad.flatMap should be consistent with Ior#flatMap" {
+            forAll { a: Int ->
+                val implicit: Semigroup<Number> = object: Semigroup<Number> {
+                    override fun combine(a: Number, b: Number): Number = a
+                }
```

Comment:
You can't create lawful monoids for `Double` and `Float` https://github.com/scalaz/scalaz/issues/334. Also to avoid confusion we should not name instances `implicits` since these are not real implicits and explicitly passed as args. I think for this anonymous instance a better name would be `intMonoidInstance` or just `IM`